src/main/java/uk/gov/hmcts/reform/sendletter/services/zip/ZipFileNameHelper.java

```diff
@@ -21,6 +22,21 @@ public static String generateName(Letter letter) {
         );
     }
 
+    public static String generateName(
+        String type,
+        String service,
+        LocalDateTime timestamp,
+        UUID id
+    ) {
+        return String.format(
+            "%s_%s_%s_%s.zip",
+            type,
+            service.replace("_", ""),
+            timestamp.format(ofPattern("ddMMyyyyHHmmss")),
+            id
+        );
+    }
```

Comment:
the existing method (that takes letter intsance) could call this one, to make sure they're kept in sync